### PR TITLE
CloudUploader: ability to specify upload error.

### PR DESCRIPTION
**Pull request by mikesherov**

### CODE REVIEW

File: Component/CloudUploader.js [hunk 1]
```diff
@@ -378,6 +379,10 @@ define([
     },
 
     _onError: function(id, name, message, xhr) {
+      if (xhr) {
+        message = this._config.messages.uploadError.replace("{file}", name);
```
Comment (mrjoelkemp) [inline Component/CloudUploader.js]: Non-blocker: Do we not have JSCS running on this to enforce single quotes?

Comment (mikesherov) [inline Component/CloudUploader.js]: no, but we should.

Comment (mikesherov) [inline Component/CloudUploader.js]: @hzoo is there an autofix for single quotes?

**Comment by mrjoelkemp** [comment]
:+1:

File: Component/CloudUploader.js (hunk 1 above)
Comment (hzoo) [inline Component/CloudUploader.js]: Yeah it should work https://github.com/jscs-dev/node-jscs/blob/08a5801fb30b8486debb70b74059a341a4ee6bf7/lib/rules/validate-quote-marks.js#L147-L154

**Comment by kaicataldo** [comment]
:+1:

**Comment by hzoo** [comment]
:+1:

**Comment by Attamusc** [comment]
:+1:

**Pull request merged**